README.md

```diff
@@ -76,31 +76,11 @@ At the moment we provide a way to automatically compile and install the library
 as a Python module. Instructions on how to compile the eFEL as a standalone C++
 library can be found [here](http://efel.readthedocs.io/en/latest/installation.html#installing-the-c-standalone-library).
 
+How to cite
+===========
+If you use this repository in your work, please refer to the "Cite this repository" button at the top of the repository page to get various citation formats, including APA and BibTeX.
```

Comment:
Why remove the citation block? It makes it harder for the users to easily access a citation they can use in bibtex. Also, what is the `cite this repository` button?

Reply:
You can also keep the original language of the citation block:

> When you use this eFEL software for your research, we ask you to cite the following publications (this includes poster presentations):

Reply:
The `cite this repository` button allows users to generate citations in various formats, including BibTeX. (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files#about-citation-files). 

This way, we only need to maintain one source for citations or do you want to keep both?

Reply:
Oh, so it will be appearing after you merge this. I was asking because I was not able to see it. Ok, you can keep your change

Reply:
I think the button should appear if you switch to this branch